src/app/pages/commission-board/rating/rating.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, Input} from '@angular/core';
+import {AppUser} from '../../../model/list/app-user';
+
+@Component({
+    selector: 'app-rating',
+    templateUrl: './rating.component.html',
+    styleUrls: ['./rating.component.scss']
```

Comment:
please add `changeDetection: ChangeDetecttionStrategy.OnPush` to ensure change detector is only called if one of the input changes, since it's the only thing that can trigger a change in this component.